clients/docs/csharp.rst

```diff
@@ -0,0 +1,132 @@
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for .NET.
+
+   .. sourcecode:: bash
+
+      cd clients/cloud/csharp/
+
+.. include:: includes/client-example-create-file.rst
+
+
+Example 1: Basic Produce & Consume
+----------------------------------
+
+In this example, the producer writes records to a topic in your |ak|
+cluster. Each record has a key representing a username (e.g. ``alice``)
+and a value of a count, formatted as json (e.g. ``{"count": 0}``). The
+consumer reads the same topic and keeps a rolling sum of the counts as
+it processes each record.
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Build the client example application
+
+   .. code:: shell
+
+       dotnet build
+
+#. Run the example application, passing in arguments for (a) whether to produce or consume (produce) (b) the topic name (c) the local file with configuration parameters to connect to your |ak| cluster and (d, Windows only) a local file with default trusted root CA certificates.
+
+   .. code:: shell
+
+      # Run the producer (Windows)
+      dotnet run produce test1 $HOME/.confluent/librdkafka.config /path/to/curl/cacert.pem
+
+      # Run the producer (other)
+      dotnet run produce test1 $HOME/.confluent/librdkafka.config
+
+   You should see:
+
+   .. code:: shell
+
+      Producing record: alice {"count":0}
+      Producing record: alice {"count":1}
+      Producing record: alice {"count":2}
+      Producing record: alice {"count":3}
+      Producing record: alice {"count":4}
+      Producing record: alice {"count":5}
+      Producing record: alice {"count":6}
+      Producing record: alice {"count":7}
+      Producing record: alice {"count":8}
+      Producing record: alice {"count":9}
+      Produced record to topic test1 partition [0] @ offset 0
+      Produced record to topic test1 partition [0] @ offset 1
+      Produced record to topic test1 partition [0] @ offset 2
+      Produced record to topic test1 partition [0] @ offset 3
+      Produced record to topic test1 partition [0] @ offset 4
+      Produced record to topic test1 partition [0] @ offset 5
+      Produced record to topic test1 partition [0] @ offset 6
+      Produced record to topic test1 partition [0] @ offset 7
+      Produced record to topic test1 partition [0] @ offset 8
+      Produced record to topic test1 partition [0] @ offset 9
+      10 messages were produced to topic test1
+
+#. View the :devx-examples:`producer code|clients/cloud/csharp/Program.cs`.
+
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the consumer, passing in arguments for (a) whether to produce or consume (consume) (b) the same topic name as used above (c) the local file with configuration parameters to connect to your |ak| cluster and (d, Windows only) a local file with default trusted root CA certificates. Verify that the consumer received all the messages:
```

Comment:
similar formatting comment as this line https://github.com/confluentinc/examples/pull/715/files#diff-bc98f71015872d118b235d29e82a31bcR63

Reply:
Ack